src/views/Dashboard/TreasuryDashboard.tsx

```diff
@@ -105,8 +133,12 @@ function TreasuryDashboard() {
       <div className="hero">
         <Box component="div" color="text.primary">
           <p>Wen (3,3) becomes (🦦,🦦)</p>
-          <h1>Welcome to Otter Kingdom</h1>
-          <h3>The Decentralized Reserve Memecoin</h3>
+          <h1>
+            <Trans i18nKey="dashboard.otterKingdom" />
```

Comment:
There are mixing use of `Trans` component and `t`. Feel we can unify these with `t`.

Reply:
Added padding

![image](https://user-images.githubusercontent.com/24249418/145313610-ff32dbc8-07f0-42bf-bcc2-16b7f1d13ed1.png)